app/models/user.rb

```diff
@@ -53,14 +54,11 @@ def documents(type: 'BirthRecord')
     []
   end
 
-  # Get the audits for this user
-  #
-  # These are audits where the user is the one who took action. Notably this
-  # doesn't include audits where an organisation views a birth record shared by
-  # this use (because the organisation takes the action). Those audits can be
-  # retrieved separately with `user.shares.map(&:audits).flatten` - see
-  # User::AuditsController#index for an implementation
-  def audits
-    Audit.where(user: self)
+  def add_role(organisation, role)
+    OrganisationMember.create!(
```

Comment:
Todo: use methods on organisation instead

Reply:
I think it is the organisation's job to add a user, not the user's job to add themselves to an organisation